tag-semantic-release/action.yaml

```
@@ -0,0 +1,96 @@
+name: 'Tag semantic release'
+description: 'Tag semantic release for your default branch'
+author: 'Andy Hsieh'
+inputs:
+  version-type:
+    required: true
+    default: 'patch'
+  github-token:
+    required: true
+  debug:
+    default: false
+  dry-run:
+    default: false
+outputs:
+  version:
+    value: ${{ steps.main.outputs.version }}
+runs:
+  using: "composite"
+  steps:
+    - run: |
+
+        echo "tsr v0.0.11"
+        export GITHUB_TOKEN=${{ inputs.github-token }}
+        git config --global user.name 'zendesk-ga'
+        git config --global user.email 'zendesk-ga@zendesk.com'
+        git config --global hub.protocol https
+
+        if [ ${{ inputs.debug }} = true ]; then
+          set -x
+        fi
+
+        if [[ ! -x "$(which hub)" ]]; then
+          echo "Missing hub cli, install it here:"
+          echo "https://github.com/github/hub"
```

Use GitHub action style error messages

```suggestion
          echo "::error::Missing hub CLI, install it here:"
          echo "::error::https://github.com/github/hub"
```